# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1292024204 28800
#      Fri Dec 10 15:36:44 2010 -0800
# Node ID cc7f30d3fd01d3f130a5eb33a1a14b7582605954
# Parent  3c7ba1e3dc21469b9f6cb16cec051b7487473ad0
speed.m: Eliminate line continuation in function definition.
This particular continuation confuses up Octave syntax mode for Vim.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-12-10  Rik  <octave@nomad.inbox5.com>
+
+	* testfun/speed.m: Eliminate line continuation in function definition.
+
 2010-12-10  Kai Habel  <kai.habel@gmx.de>
 
 	* plot/private/__pie__.m: Add missing slice option to pie and
 	pie3 function.
 	* plot/pie.m, plot/pie3.m: Add new demo.
 
 2010-12-10  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
diff --git a/scripts/testfun/speed.m b/scripts/testfun/speed.m
--- a/scripts/testfun/speed.m
+++ b/scripts/testfun/speed.m
@@ -144,18 +144,17 @@
 ##
 ## Type @code{example('speed')} to see some real examples.  Note that for 
 ## obscure reasons, examples 1 and 2 can not be run directly using 
 ## @code{demo('speed')}.  Instead use, @code{eval ( example('speed', 1) )}
 ## or @code{eval ( example('speed', 2) )}.
 ## @end deftypefn
 
 ## FIXME: consider two dimensional speedup surfaces for functions like kron.
-function [__order, __test_n, __tnew, __torig] ...
-      = speed (__f1, __init, __max_n, __f2, __tol)
+function [__order, __test_n, __tnew, __torig] = speed (__f1, __init, __max_n, __f2, __tol)
 
   if (nargin < 1 || nargin > 6)
     print_usage ();
   endif
 
   if (nargin < 2 || isempty (__init))
     __init = "x = randn(n, 1);";
   endif
